metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/GenerateSchemaCommand.java

```diff
@@ -177,13 +177,13 @@ protected ExitStatus executeCommand(
     if (destination != null) {
       if (Files.exists(destination)) {
         if (!cmdLine.hasOption(OVERWRITE_OPTION)) {
-          return ExitCode.FAIL.exitMessage( // NOPMD readability
+          return ExitCode.INPUT_ERROR.exitMessage( // NOPMD readability
               String.format("The provided destination '%s' already exists and the '%s' option was not provided.",
                   destination,
                   OptionUtils.toArgument(OVERWRITE_OPTION)));
         }
         if (!Files.isWritable(destination)) {
-          return ExitCode.FAIL.exitMessage( // NOPMD readability
+          return ExitCode.INPUT_ERROR.exitMessage( // NOPMD readability
```

Comment:
Minor, non-blocking. If you like the above change:

```suggestion
          return ExitCode.IO_ERROR.exitMessage( // NOPMD readability
```